db/structure.sql

```diff
@@ -1609,6 +1609,7 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20180508131152'),
 ('20180517140929'),
 ('20180522132456'),
+('20180522132457'),
```

Comment:
There is no migration in this branch ... is there a migration missing from `master`?

Reply:
Hmmmm no idea, this migration isn't in my branch either 👻spooky!! I'll remove it